### ISSUE TITLE
about jpg image quality

### ISSUE THREAD

**Issue by suke27** [xinntao/BasicSR#96]
hello, as I know gan method can not work well for jpg images. but I heard ESRGAN with RAGAN resolve this problem(Bell effect). is that true?

**Comment by xinntao** [comment]
No, ESRGAN still has problems with jpg images. Because ESRGAN does not have the ability to remove the compression artifacts and thus just enlarges these artifacts.

**Comment by young666** [comment]
@xinntao Could I convert the JPGs to PNGs using Matlab? Sorry if this question is quite stupid 😄

**Comment by xinntao** [comment]
Of course. I prefer to write a python script to convert. 
But it only converts the format. If you have jpg images with obvious compression artifacts, these artifacts will be kept even though you convert them to png formats. 
You may need to run some DeCompression model to remove jpg compression artifacts.

**Comment by mygmyg** [comment]
@xinntao hello, what is the possible reason that gan does not have the ability to remove the compression artifacts, while  just use l1 loss as srcnn can remove the compression artifacts ,but  have cartoon style problem ?

**Comment by suke27** [comment]
i think it is caused by gan loss, just like perception loss will cause grid texture.

**Comment by xinntao** [comment]
@mygmyg  Because we do not train the network to remove JPEG artifacts. We can also use training dataset with JPEG artifacts as inputs and train the network. I think the learned network will have ability to remove the JPEG artifacts.
